src/mono/sample/iOS-NativeAOT/Program.csproj

```diff
@@ -0,0 +1,96 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <OutputPath>bin</OutputPath>
+    <IntermediateOutputPath>$(MSBuildThisFileDirectory)/obj/</IntermediateOutputPath>
+    <TargetFramework>net7.0</TargetFramework>
+    <TargetOS Condition="'$(TargetOS)' == ''">ios</TargetOS>
+    <TargetOS Condition="'$(TargetsiOSSimulator)' == 'true'">iossimulator</TargetOS>
+    <DeployAndRun Condition="'$(DeployAndRun)' == ''">true</DeployAndRun>
+    <RuntimeIdentifier>$(TargetOS)-$(TargetArchitecture)</RuntimeIdentifier>
+    <AppName>HelloiOS</AppName>
```

Comment:
Nit: Might be good to alter the name to indicate that it is a nativeaot sample app. Useful when working on the same device with multiple engines (mono and nativeaot).

Reply:
Good point. Although I think for comparisons between compilers/runtimes it makes sense to compare against the same app "name". Will keep this in mind.